src/reactviews/pages/QueryResult/queryResultSetup.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Theme, webLightTheme } from "@fluentui/react-components";
+
+export const saveAsCsv = (theme: Theme) => {
```

Comment:
the const name is a bit confusing,

we can probably do something like

```
export const saveAsCsvIcon = (theme: Theme) => theme === webLightTheme ? ... : ...
```

Reply:
Or better yet, we can move all icon usage in webviews to a shared `Icons` class to remove both the confusion and the verbose naming (and maybe even the theme selection?)